### ISSUE TITLE
twisted.internet.task.react is missing a @since epydoc tag

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5990]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#5990|
|Type|defect|
|Created|2012-09-16 00:53:15Z|

The new `react` API should document in what version it was added.

<details><summary>Searchable metadata</summary>

```
trac-id__5990 5990
type__defect defect
reporter__itamar itamar
priority__lowest lowest
milestone__Twisted_12_3 Twisted-12.3
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1347756795000000 1347756795000000
changetime__1355417183000000 1355417183000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

_All_ APIs should have this.  But dealing with them one by one strikes me as an incredible waste of time as well as highly prone to failure.

See [#5309](https://github.com/twisted/twisted/issues/5309).

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve set status to `closed`|
|-|-|

Fixed with the following commit:

In [36544]) Merge react-exit-6159

Author: therve Reviewer: exarkun Fixes: [#6159](https://github.com/twisted/twisted/issues/6159)

Change twisted.internet.task.react to exit the python process with non-0 exit code when the deferred returned failed.
